server/channels/app/custom_profile_attributes.go

```diff
@@ -12,7 +12,10 @@ import (
 	"github.com/pkg/errors"
 )
 
-const CustomProfileAttributesFieldLimit = 20
+const (
+	CustomProfileAttributesFieldLimit    = 20
+	CustomProfileAttributesValuesPerPage = CustomProfileAttributesFieldLimit
```

Comment:
nit: I think this property may be confusing to the reader as we're creating a `const` explicitly for paging (which is something that the feature doesn't do) instead of using the `limit` where a page is required.